libr/bin/demangle.c

```diff
@@ -165,6 +165,13 @@ R_API char *r_bin_demangle_objc(RBinFile *binfile, const char *sym) {
 	char *args = NULL;
 	int i, nargs = 0;
 	const char *type = NULL;
+
+	if (binfile && binfile->o) {
+		if (binfile->o->classes) {
```

Comment:
you can merge this condition with the previous one:
binfile && binfile->o && binfile->o->classes